src/repo/github.go

```diff
@@ -0,0 +1,51 @@
+package repo
+
+import (
+	"fmt"
+	"net/http"
+	"bytes"
+	"time"
+	"encoding/json"
+)
+
+type TokenIssuer interface {
+	GeneratedJwtToken() (string, error)
+}
+type GithubClient struct {
+	tokenIssuer TokenIssuer
+}
+func NewGithubClient(tokenIssuer TokenIssuer) *GithubClient {
+	return &GithubClient{
+	  tokenIssuer: tokenIssuer,
+	}
+  }
+
+func (c *GithubClient)  IssueAccessToken(installationID string) (string,error) {
+	jwtToken,err := c.tokenIssuer.GeneratedJwtToken()
+	if(err != nil){
+	return "",fmt.Errorf("%s",err)
+	}
+	url :=fmt.Sprintf("https://api.github.com/app/installations/%s/access_tokens",installationID)
+	req, err := http.NewRequest("POST", url,bytes.NewBuffer(nil))
```

Comment:
didn't check this error.
also you can use `nil` instead of bytes.NewBuffer(nil).

Reply:
done